code/modules/power/apc/apc.dm

```diff
@@ -488,6 +491,9 @@
 	if(issilicon(user))
 		var/mob/living/silicon/ai/AI = user
 		var/mob/living/silicon/robot/robot = user
+		if(malfai == "DVORAK") //I'm not making an actual ai core for this, you get a string. Naming an AI DVORAK will not do anything, as it will be an actual AI core, instead of a string
```

Comment:
Are you setting this malfai typed var to a string? Not a huge fan of that

Reply:
Open for a better solution that isn't spawning a mindless AI mob to link to every APC

Reply:
solution found, testing then pushing